eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -160,6 +160,12 @@ jobs:
 
       - ${{ parameters.PostSteps }}
 
+      - script: echo 'TestType is $(variables['TestType']}}'
```

Comment:
I assume this is just for temp debugging?

Reply:
If so, the syntax needs to be as below. The way you are using `${{ }}` won't work, as those template expressions are resolved before devops populates the variable values.

```
      - script: echo 'TestType is $(TestType)'
```

Reply:
yes, I'm just trying to make sure we send the correct value to the argument.
this PR is just to run the pipeline and test it. I will not merge it.
and thanks for the info.